### PR TITLE
Release candidate

### PR DESCRIPTION
**Pull request by yarneo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,94 @@
+# 72.2.0
+
+In this major release we added scrimColor support for MDCBottomSheetController and added a theming extension support for MDCAlertController. We also added TextFields screenshot testing along with various bug fixes.
```
Comment (romoore) [inline CHANGELOG.md:3]: Should this be minor? Or should it be version 73.0.0?

Comment (romoore) [inline CHANGELOG.md:3]: Nit: s/screenshot/snapshot/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,94 @@
...
+
+## New features
+
+### Bottom Sheet scrimColor API usage example
+```
```
Comment (romoore) [inline CHANGELOG.md:8]: Tip: Nit: please mark the languages 

\```objc
...

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,94 @@
...
+  MDCBottomSheetController *bottomSheet =
+      [[MDCBottomSheetController alloc] initWithContentViewController:viewController];
+  bottomSheet.scrimColor = UIColor.blueColor;
+  [self presentViewController:bottomSheet animated:YES completion:nil];
+```
+
+### Dialogs Theming Extension usage example
+```
```
Comment (romoore) [inline CHANGELOG.md:16]: Tip: Nit: Please mark the language 

\```swift
...

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,94 @@
...
+* [Remove clipping view (#5897)](https://github.com/material-components/material-components-ios/commit/be33ef513ede24882ee04fc7cda122e7a68677b2) (Cody Weaver)
+
+### private/KeyboardWatcher
+
+* [fix keyboard watcher not taking into regards compatibility mode (#5957)](https://github.com/material-components/material-components-ios/commit/39a2a282f5070ae51bfa42fe029b3b232756f01e) (Yarden Eitan)
+
+## Multi-component changes
+
+* [Add FullWidth snapshot tests. (#5953)](https://github.com/material-components/material-components-ios/commit/ab7a6467db5edb7dc87de589fa09e073db8e8d65) (Robert Moore)
+* [Add Outlined snapshot tests (#5939)](https://github.com/material-components/material-components-ios/commit/495a29ca69305acde9dff9d1c67f7939011548b1) (Robert Moore)
```
Comment (romoore) [inline CHANGELOG.md:75]: This was for `TextFields`, not multiple components.

**Review by romoore — COMMENTED** [review]
CHANGELOG needs some improvement.

Based on the release message, it's not clear if this is a major (breaking) or minor change.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,94 @@
+# 72.2.0
+
+In this major release we added scrimColor support for MDCBottomSheetController and added a theming extension support for MDCAlertController. We also added TextFields screenshot testing along with various bug fixes.
+
+## New features
```
Comment (romoore) [inline CHANGELOG.md:5]: Add section?

```
## Bug fixes

{Git-LFS} Exclude git-lfs from `stable` #5977
```

**Review by romoore — APPROVED** [review]
LGTM! Release all the things!

**Review by randallli — APPROVED** [review]

**Pull request merged**